### ISSUE TITLE
Should Safari 15.4 have its own entry?

### ISSUE THREAD

**Issue by JohnRiv** [compat-table/compat-table#1826]
I noticed both [`.at()` method on the built-in indexables](https://github.com/tc39/proposal-relative-indexing-method/) and [`Object.hasOwn`](https://github.com/tc39/proposal-accessible-object-hasownproperty) were added to Safari in v15.4. It's unclear looking at the current table that the currently supported Safari version supports those.

I tested it to confirm the support is there, and it is also confirmed by:
- [Apple's release notes](https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes#JavaScript)
- MDN:
  - [Array.prototype.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)
  - [String.prototype.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/at)
  - [TypedArray.prototype.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/at#browser_compatibility)
  - [Object.hasOwn()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn#browser_compatibility)

**Comment by ljharb** [comment]
Yes, whenever a new feature is added to an env, we add a new entry to differentiate.

As such, https://github.com/kangax/compat-table/blob/9742598c5883018d68885e766840b4c5d7efab8f/environments.json#L3264-L3277 should be extended to include safari15_3, and safari15_4, so that "safari15" points to the latest, and we can then identify for these features "the last version that didn't have it" and "the first version that did".

**Comment by JohnRiv** [comment]
I was going through my old opened GitHub issues and noticed this was fixed
